### DISCUSSION TITLE
Q: Is there a way to link two files together for always opening them togehter?

### DISCUSSION THREAD

**Discussion by autoca** [microsoft/vscode-discussions#234] [Extension Development QnA] [answered]
Hi all,

for example, a certain html file and a certain css file.
I would want them to always open together side by side.
If I open one or the other then the 2nd file should automatically be opened as well in split view in an editor on the other pane.
When one or both are already open but not active, and I activate one of their editors then the editor of the other file on the other pane should automatically also be moved to the foreground or be opened if not already open.

Maybe this could be viewed as some kind of "Mini-Workspace" inside a normal Workspace.

Is this already possible? I searched Google for a while but it seems Google does not understand or this is not yet possible.

MTIA and regards, autoca

**Comment by alefragnani** [comment, marked as answer]
Hi @autoca ,

No, there is no such feature, nor API to automatically accomplish this.

However, there are a few events available that you may listen to develop an extension, like `onDidOpenTextDocument`, `onDidChangeActiveTextEditor` and `onDidChangeVisibleTextEditors` at least. I think it's a good starting point.

Hope this helps

**Reply by autoca** [reply]
Ok, thank you for your reply.
